lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -192,6 +192,19 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     element.appendChild(passedElem);
     return element;
   }
+
+  /**
+   * @param {LH.ReportResult.Category} category
+   * @return {null|string}
+   * @override
+   */
+  static getLastScreenshot(category) {
+    const auditRef = category.auditRefs.find(audit => audit.id === 'screenshot-thumbnails');
```

Comment:
should we give them a new proper full-res final screenshot audit?

Reply:
done.